### PR TITLE
Added Dawn profile for Gnome Terminal

### LINKED ISSUE #2: Adding the DAWN theme would be nice
It would be nice to also port the Dawn palette theme to Gnome Terminal

### PR DESCRIPTION
**Pull request by diorcula**
- still needs testing as this is my first ever contribution to a gnome profile theme.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <p align="center">
     <img src="https://github.com/rose-pine/rose-pine-theme/raw/main/assets/icon.png" width="80" />
-    <h2 align="center">Rosé Pine for GNOME Terminal</h2>
+    <h2 align="center">Rosé Pine Dawn for GNOME Terminal</h2>
```
Comment (mvllow) [inline readme.md]: The dawn variant should be in addition, not instead of the base variant

**Review by mvllow — CHANGES_REQUESTED** [review]
Thank you for taking the time to contribute this. Please add Rosé Pine dawn as a separate file with only relevant parts of that theme. Our base theme does have some unnecessary bits which you pointed out but that should be another PR 😌

If it's any easier, feel free to revise the PR to only add `rosepine-dawn.dconf` and we can update the readme afterwards

File: readme.md (hunk 1 above)
Comment (diorcula) [reply readme.md]: sorry i forgot about this! i fixed it and will make a new PR!

**Comment by mvllow** [comment]
Thank you!

**Pull request merged**